lib/index.js

```diff
@@ -578,6 +578,26 @@ Object.defineProperties( module.exports, {
     enumerable: true,
     configurable: false,
     writable: true
-  }
+  },
+
+  getElasticSearchCache: {
+    /**
+     * Returns ElasticSearch class constructor
+     *
+     * @param client - override elasticsearch client
+     * @param ttl - override TTL value
+     * @returns {Object}
+     */
+    value: function getElasticSearchCache(client, ttl) {
+      let ElasticSearchCache = require('./elasticsearch-cache');
+      return new ElasticSearchCache({
```

Comment:
@fqjony It overwrites default constructor `options` if `client` and `ttl` even are not provided here.

Reply:
I would recommend to use `options` object as parameter. And set its default value: `options = {}`.

```
function getElasticSearchCache(options = {})
```